### ISSUE TITLE
Partial patches

### ISSUE THREAD

**Issue by bcssov** [bcssov/IronyModManager#57]
From discord:

> - Making versions of collections or modifying a collection someone else distributed is a good use case but there's not much support today, even changing the name of a collection seems to be difficult (I've tried changing the zip and collection name inside it but get an exception on load, I assume there are other names/paths that must point to the same place I failed to replace)
> - Paradoxos takes a little different angle on conflicts by looking at them from a "mod-centric" point of view, Irony focuses on the structure / overrides instead, this is great for being exhaustive /perfectionist, but doesn't help authors take "shortcuts" when they know (for example) that the graphics don't conflict or that they have no interest in localization patches
> - Thanks to the approach above, you actually have the views and raw materials you need to make "partial patches" from collections, i.e. create a patch that is just for graphics, a patch just for game mechanics, a patch just for traits, etc. but there's no features to help this workflow out (you'd want to be able to make a new collection from another one, and probably mark a conflict "resolved" so that the next partial-patch knows it's handled); if Irony could do this (plus maybe merge mods and compare to base/vanilla) I don't know why anyone would use any other manager for paradox mods/collection creation
> tl;dr I think the exhaustive/perfectionist structure centric conflict resolution Irony has could be best of both worlds if you had a way to make partial patches

**Comment by bcssov** [comment]
This more or less can be accomplisted with various duplicate\rename\mod filter\git ignore like functionality. Closing.
